manager/integration/deploy/minio-backupstore.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: minio-secret
+  namespace: longhorn-system
+type: Opaque
+data:
+  AWS_ACCESS_KEY_ID: bG9uZ2hvcm4tdGVzdC1hY2Nlc3Mta2V5 # longhorn-test-access-key
+  AWS_SECRET_ACCESS_KEY: bG9uZ2hvcm4tdGVzdC1zZWNyZXQta2V5 # longhorn-test-secret-key
+  AWS_ENDPOINTS: aHR0cDovL21pbmlvLXNlcnZpY2UubG9uZ2hvcm4tc3lzdGVtOjkwMDA= # http://minio-service.longhorn-system:9000
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: longhorn-test-minio
+  namespace: longhorn-system
+  labels:
+    app: longhorn-test-minio
+spec:
+  volumes:
+  - name: minio-volume
+    emptyDir: {}
+  containers:
+  - name: minio
+    image: minio/minio
+    command: ["sh", "-c", "mkdir -p /storage/testbucket/testpath && exec /usr/bin/minio server /storage"]
```

Comment:
We can call them `backupbucket` and `backupstore`. We would reuse the yaml to deploy a simple backupstore for user.

Also, I am curious what would happen if we don't create the `testpath`(`backupstore`) here? If we only create the bucket, will it work for the engine?